systems/plants/@RigidBodyManipulator/forwardKin.m

```diff
@@ -32,27 +32,41 @@
 % x will be a 7xm matrix and (following out gradient convention) J will be
 % a ((7xm)*(q)) matrix with [J1;J2;....;Jm] where Ji = dxidq
 
+if nargin < 7
+  options = struct();
+  if nargin > 4
+    options.rotation_type = rotation_type;
+  end
+else
+  % if options struct is passed in, make sure that rotation_type argument
+  % is empty to avoid conflicts
+  valuecheck(rotation_type, []);
```

Comment:
good.  i'd say let's zap the rotation_type input (and definitely this valuecheck)